build/configs/esp_wrover_kit/tash_wlan/defconfig

```diff
@@ -778,8 +781,18 @@ CONFIG_MTD=y
 CONFIG_MTD_PARTITION=y
 CONFIG_MTD_PARTITION_NAMES=y
 # CONFIG_MTD_PROGMEM is not set
-# CONFIG_MTD_FTL is not set
-# CONFIG_MTD_CONFIG is not set
+CONFIG_MTD_FTL=y
+
+#
+# MTD_FTL Configurations
```

Comment:
is it necessary to enable fs in tash_wlan config? maybe need another config dir?

Reply:
yes, wifi utc and wm_test will save basic config info.
like current ap info to connect for station.